components/modal/confirm.tsx

```diff
@@ -76,18 +71,34 @@ export default function confirm(config) {
     [`${prefixCls}-${props.type}`]: true,
   }, props.className);
 
+  const dialogProps = assign({
```

Comment:
Sorry. What about this?

```js
const defaultProps = {
  visible: true,
  title: '',
  footer: '',
  transitionName: 'zoom',
  maskTransitionName: 'fade',
  maskClosable: false, // 默认为 false，保持旧版默认行为
  style: {},
  width: 416,
};
const dialogProps = omit(props, [
  'type',
  'prefixCls',
  'className',
  'iconType',
  'title',
  'content',
  'okCancel',
  'okText',
  'cancelText',
  'onOk',
  'onCancel',
]);

ReactDOM.render(
  <Dialog
    {...defaultProps}
    {...dialogProps}
```

And, what exactly is the "old style"?

I see an old issue https://github.com/ant-design/ant-design/issues/4488. If you have resolved that problem completely, these issue and pr are not created. So, our time is saved.